server/src/main.py

```diff
@@ -28,6 +28,10 @@ def init_app_without_routes(disable_csrf=False):
 
     global db
     global migrate
+    class SQLAlchemy(_BaseSQLAlchemy):
```

Comment:
`_BaseSQLAlchemy` needs to be set as an alias for `SQLAlchemy` during the import

Reply:
Knew I forgot something...